tox.ini

```diff
@@ -21,7 +21,7 @@ commands = flake8 --max-complexity=8 pipeline --exclude='test/output' test
 [testenv:pylint-errors]
 deps = pylint
        -r{toxinidir}/requirements.txt
-commands = pylint -f colorized -E pipeline --ignore test/output test
+commands = pylint -f colorized -E pipeline
```

Comment:
Note: per @michaelbausor we probably were never actually linting this directory, because when you add an `__init__.py` to fix [the build error here](https://travis-ci.org/googleapis/artman/builds/144226454), it produces even more lint errors.